index.css

```diff
@@ -504,3 +563,346 @@ textarea {
+    background-repeat: no-repeat;
+    margin-top: 90px;
+  }
+
+  .box3 {
+    height: 435px;
+    background: url(../Desktop/Placeholder4.png);
+    background-repeat: no-repeat;
+    margin-top: 90px;
+  }
+
+  .box4 {
+    height: 435px;
+    background: url(../Desktop/Placeholder5.png);
+    background-repeat: no-repeat;
+    margin-top: 90px;
+  }
+
+  .box {
+    height: 435px;
+    background: url(../Desktop/Placeholder3.png);
+    background-repeat: no-repeat;
+    margin-top: 90px;
+  }
+
+  .dashbord1 {
+    display: none;
+  }
+
+  .texto1 {
+    display: none;
+  }
+
+  .contenido {
+    display: grid;
+    grid-template-columns: repeat(3, 400px);
+    grid-gap: 10px;
+    background: url(Desktop/gymfit.png);
+    background-repeat: no-repeat;
+    background-position: 15% 0%;
+  }
+
+  .multi {
+    height: 48px;
+    font-family: "Crete Round", serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 40px;
+    line-height: 52px;
+  }
+
+  .brian1 button {
+    width: 369px;
+    height: 60px;
+    margin-top: 294px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: bold;
+    background-color: #ff6b00;
+    font-size: 17px;
+    color: #fff;
+  }
+
+  .brian button {
+    display: none;
+  }
+
+  .dashbord {
+    margin-top: 130px;
+  }
+
+  .dashbord2 {
+    margin-top: 130px;
+  }
+
+  .texto3 {
+    width: 337px;
+    height: 50px;
+    margin: 0;
+  }
+
+  .botones3 button {
+    width: 95px;
+    height: 40px;
+  }
+
+  .about {
+    background: #1c1a19 url(Desktop/Frame.png) 100% 40% no-repeat;
+    margin-top: 80px;
+    height: 610px;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+  }
+
+  .about h3 {
+    width: 466px;
+    height: 78px;
+    font-family: "Crete Round", serif;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 72px;
+    line-height: 88px;
+    color: #ff6b00;
+    padding-top: 90px;
+  }
+
+  .about p {
+    width: 566px;
+    height: 148px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    color: #f4f5f7;
+    margin: 30px 0 0 90px;
+  }
+
+  .proyect2 button {
+    width: 161px;
+    height: 48px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 17px;
+    line-height: 24px;
+    background: #ff6b00;
+    color: #fff;
+    margin-left: 90px;
+  }
+
+  .experience {
+    display: grid;
+    grid-template-columns: repeat(3, 400px);
+    height: 500px;
+  }
+
+  .letras2 {
+    margin-left: 20px;
+  }
+
+  .agrupador {
+    text-align: start;
+  }
+
+  .agrupador2 {
+    text-align: start;
+  }
+
+  .agrupador3 {
+    text-align: start;
+    background-position: 8% 6%;
+  }
+
+  .formato {
+    height: 500px;
+    margin: 200px 0 0 0;
+  }
+
+  footer {
+    display: grid;
+    grid-template-columns: repeat(2, 650px);
+  }
+
+  #msg {
```

Comment:
- [ ] Kindly make sure that the social icons container is properly aligned to the center.
your design
![Capture2](https://user-images.githubusercontent.com/85095702/160022856-be0bd26f-e73a-4702-87eb-b45680b5f836.PNG)